src/components/designer/default/NetworkDesignerTab.tsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import { CloudSyncOutlined, DownOutlined, UpOutlined } from '@ant-design/icons';
+import styled from '@emotion/styled';
+import { Button } from 'antd';
+import { usePrefixedTranslation } from 'hooks';
+import { NodeImplementation } from 'shared/types';
+import { useStoreActions, useStoreState } from 'store';
+import { dockerConfigs } from 'utils/constants';
+import { getPolarPlatform } from 'utils/system';
+import DraggableNode from './DraggableNode';
+
+const Styled = {
+  AddNodes: styled.h3`
+    margin-top: 30px;
+  `,
+  AddDesc: styled.p`
+    opacity: 0.5;
+    font-size: 12px;
+  `,
+  Nodes: styled.div`
+    display: flex;
+    flex-direction: column;
+  `,
+  NodeRow: styled.div`
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  `,
+  Arrow: styled.span<{ visible?: boolean; collapsible?: boolean }>`
+    margin-top: ${props => (props.visible ? 20 : 0)}px;
+    width: 25px;
+    text-align: right;
+
+    > .anticon {
+      opacity: 0.4;
+      display: ${props => (props.collapsible ? 'inline' : 'none')};
+      cursor: pointer;
+
+      &:hover {
+        opacity: 1;
+      }
+    }
+  `,
+  UpdatesButton: styled(Button)`
+    margin-top: 30px;
+  `,
+};
+
+const NetworkDesignerCard: React.FC = () => {
+  const { l } = usePrefixedTranslation(
+    'cmps.designer.default.DefaultSidebar.NetworkDesignerTab',
+  );
```

Comment:
- The component name should be `NetworkDesignerTab`
- The translation prefix should be `cmps.designer.default.NetworkDesignerTab`